src/user-card/card.tsx

```diff
@@ -56,39 +72,28 @@ export default class UserCard extends PureComponent<UserCardProps> {
       unverifiedEmail: PropTypes.bool
     }).isRequired,
 
-    wording: PropTypes.shape({
-      banned: PropTypes.string.isRequired,
-      online: PropTypes.string.isRequired,
-      offline: PropTypes.string.isRequired,
-      copyToClipboard: PropTypes.string,
-      copiedToClipboard: PropTypes.string,
-      copingToClipboardError: PropTypes.string,
-      unverified: PropTypes.string
-    })
+    wording: translationsShape,
+    translations: translationsShape
   };
 
-  static defaultProps = {
-    wording: {
-      banned: 'banned',
-      online: 'online',
-      offline: 'offline',
-      copyToClipboard: 'Copy email to clipboard',
-      copiedToClipboard: 'Email was copied to clipboard',
-      copingToClipboardError: 'Failed to copy to clipboard',
-      unverified: 'Unverified'
-    }
-  };
+  static contextType = I18nContext;
 
   copyEmail = () => {
-    const wording = {...UserCard.defaultProps.wording, ...this.props.wording};
+    const messages = this.context as Messages;
+    const messageOverrides = this.props.wording || this.props.translations;
+
+    const wording = messageOverrides ?? messages;
     clipboard.copyText(
       this.props.user.email || '', wording.copiedToClipboard, wording.copingToClipboardError
     );
```

Comment:
We can already get rid of the term `wording` within the component:
```suggestion
    const messages = this.props.wording ?? this.props.translations ?? this.context;

    clipboard.copyText(
      this.props.user.email || '', messages.copiedToClipboard, messages.copingToClipboardError
    );
```
Once we have deleted the `wording` prop, it will be sufficient to simplify the expression for calculating the value of the `messages`:
```ts
    const messages = this.props.translations ?? this.context;
```

Reply:
If we delete "wording" prop, that would be breaking change, that's why we have to keep it for a while

Reply:
I'm not removing the `wording` prop in my suggestion. Sorry for the confusion. 

> Once we have deleted the wording prop, it will be sufficient to simplify the expression for calculating the value of the messages:

It was about the future major version of the library. But we can now reduce the use of `wording` in this component. That was my suggestion.

Reply:
Got it!

I believe it is moreless OK now in a new PR 5542, how do you think?

https://github.com/JetBrains/ring-ui/blob/6d16d099f9ca352f5c88000e7a0677dfb4faed4e/src/user-card/card.tsx#L82

Reply:
In my opinion it's better to use something like this:
```ts
const translations = this.props.translations || this.props.wording; 
```
because `translations` is a canonical term in the codebase, but this is a very minor point, so let's leave it as that :)